packages/manager/modules/vrack/src/partials/available.html

```diff
@@ -28,23 +28,23 @@ <h2 class="vrack__section-title" data-translate="vrack_your_services"></h2>
                 'ovh-font-vRack': serviceType === 'legacyVrack'}"
             >
             </i>
-            {{ ::VrackCtrl.getDisplayName(serviceType) }}
+            {{ ::$ctrl.getDisplayName(serviceType) }}
```

Comment:
I know you've just rename the controller, but this can be in a  :
```suggestion
           <span data-ng-bind="::$ctrl.getDisplayName(serviceType)"></span>
```

Reply:
This is fixed in all places. Thanks :)